File: SoloLearn Codes/It's a Sign/ItsASign.java
import java.util.Arrays;
import java.util.Scanner;

public class ItsASign
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(Arrays
                .stream(new String[4])
                .map(w -> sc.nextLine().trim())
                .anyMatch(w -> w.equals(new StringBuilder(w)
                        .reverse()
                        .toString())) ? "Open" : "Trash");
    }
}

File: SoloLearn Codes/It's a Sign/ItsASign.cs
using System;
using System.Linq;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(new int[4]
                .Select(i => Console.ReadLine())
                .Any(i => Equals(i, string.Concat(Enumerable.Reverse(i)))) ? "Open" : "Trash");

        }
    }
}
